### ISSUE TITLE
(hexchat:2717): WARNING **: 16:52:34.340: Error parsing translated event Receive Wallops Loading untranslated.

### ISSUE THREAD

**Issue by tulliana** [hexchat/hexchat#2165]
I open hexchat and I get this warning.

```
(hexchat:2717): WARNING **: 16:52:34.340: Error parsing translated event Receive Wallops
Loading untranslated.
```
![hexchat-issue](https://user-images.githubusercontent.com/2324162/38198765-bc3f9bf2-3696-11e8-9a9b-aae078e5cc8f.png)

**Comment by TingPing** [comment]
What language?

**Comment by tulliana** [comment]
Turkish (tr_TR)

**Comment by TingPing** [comment]
Fixed by 5cd70622aaabc0419cc0b547808f09221530f0fc
